### PR TITLE
Add MercurySlider

### PR DESCRIPTION
**Pull request by sath-eesh**

### CODE REVIEW

File: Sources/Assets.xcassets/BottomProgressbg.imageset/Contents.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "filename" : "BottomProgressbg.svg",
```
Comment (jayahariv) [inline Sources/Assets.xcassets/BottomProgressbg.imageset/Contents.json]: update the file name to sliderBackground

File: Sources/Assets.xcassets/TopPrgressviewbg.imageset/Contents.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "filename" : "TopPrgressviewbg.svg",
```
Comment (jayahariv) [inline Sources/Assets.xcassets/TopPrgressviewbg.imageset/Contents.json]: name it appropriately, topBackgroundSlider, bottomBackgroundSlider, something like that?

File: Sources/Assets.xcassets/progressthumb.imageset/Contents.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "filename" : "progressthumb.svg",
```
Comment (jayahariv) [inline Sources/Assets.xcassets/progressthumb.imageset/Contents.json]: sliderThumb ?

File: Sources/Mercury/Slider/MercurySlider.swift [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+        self.config = config
+    }
+    
+    public var body: some View {
+        VStack {
+            Text(text)
+                .font(.custom(config.primaryFont.name, size: config.primaryFont.size))
+                .foregroundColor(config.primaryColor)
+            HStack {
+                Text("Low")
```
Comment (jayahariv) [inline Sources/Mercury/Slider/MercurySlider.swift]: get the value from caller

File: Sources/Mercury/Slider/MercurySlider.swift [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+                    .font(.custom(config.secondaryFont.name, size: config.secondaryFont.size))
+                    .foregroundColor(config.secondaryColor)
+                    .padding(.leading, 16)
+                Spacer()
+                Text("High")
```
Comment (jayahariv) [inline Sources/Mercury/Slider/MercurySlider.swift]: same get from caller?

File: Sources/Mercury/Slider/MercurySlider.swift [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+            RoundedRectangle(cornerRadius: 5)
+                .foregroundColor(MercuryColor.rectangle2.swiftUIColor)
+                .frame(width: (UIScreen.main.bounds.width - 32) * CGFloat(progress), height: 10)
+            
+            // Thumb Image
+            Image("progressthumb")
+                .aspectRatio(contentMode: .fit)
+                .frame(width: 20, height: 20)
+                .padding(.top, 4)
+                .offset(x: (UIScreen.main.bounds.width - 32) * CGFloat(progress) - 7)
```
Comment (jayahariv) [inline Sources/Mercury/Slider/MercurySlider.swift:99]: what is this 32?

Comment (sath-eesh) [reply Sources/Mercury/Slider/MercurySlider.swift:99]: No idea🤧. I just copy -> paste from the main code.

File: Sources/Mercury/Slider/MercurySlider.swift [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+//  Created by Satheesh on 28/02/24.
+//
+
+import Foundation
+import SwiftUI
+
+public struct MercuryConfig {
+    var primaryText: String
+    var secondaryText1: String
+    var secondaryText2: String
```
Comment (jayahariv) [inline Sources/Mercury/Slider/MercurySlider.swift]: leftLabel & rightLabel ?

File: Sources/Mercury/Slider/MercurySlider.swift (hunk 6 above)
Comment (jayahariv) [reply Sources/Mercury/Slider/MercurySlider.swift:99]: 16 * 2 ?

**Review by jayahariv — APPROVED** [review]

**Review by jayahariv — CHANGES_REQUESTED** [review]
minor feedback

**Review by jayahariv — APPROVED** [review]

**Pull request merged**